_devSpecs/Tool.html

```diff
@@ -29,44 +29,87 @@
     such tools in Web pages and applications. The goal of the specification is to
     make it easier to discover, exchange and integrate information about life science
     tools across the Internet.
-  version: "0.2-draft"
-  version_date: 20180821T104717
-  official_type: http://semanticscience.org/resource/SIO_000097
+  version: 0.3-DRAFT
+  version_date: 20181121T214754
+  official_type: http://schema.org/SoftwareApplication
   full_example: https://github.com/BioSchemas/specifications/tree/master/Tool/examples/
 mapping:
-- property: alternateName
+- property: additionalType
   expected_types:
   - Text
```

Comment:
As terms from a controlled vocabulary should be used, would it not be better to have DefinedTerm as expected range?